README.md

```diff
@@ -192,6 +193,20 @@ runtime 'io.opencensus:opencensus-impl:0.15.0'
 If the application owner wants to export in-process tracing and stats data via HTML debugging pages
 see this [link](https://github.com/census-instrumentation/opencensus-java/tree/master/contrib/zpages#quickstart).
 
+## Versioning
+  
+This library follows [Semantic Versioning][semver].
+  
+**GA**: Libraries defined at a GA quality level are stable, and will not introduce 
+backwards-incompatible changes in any minor or patch releases. We will address issues and requests 
+with the highest priority.
+  
+**Beta**: Libraries defined at a Beta quality level are expected to be mostly stable and we're 
+working towards their release candidate. We will address issues and requests with a higher priority.
```

Comment:
You could say that we may introduce backwards incompatible changes in a minor version release, though not in a patch release.  You could also describe the deprecation process, though I'm not sure whether we've decided on a concrete process for versions before 1.0.  I think we've been following something like this:

If an element is part of an API that is only meant to be used by exporters or other opencensus libraries, then there is no deprecation period.  Otherwise, we will deprecate it for 1.5 years before removing it, if possible.